apps/etterlatte-saksbehandling-ui/client/src/components/behandling/brevutfall/BrevutfallSkjema.tsx

```diff
@@ -148,17 +172,17 @@ export const BrevutfallSkjema = ({
               legend={<EtterbetalingHjelpeTekst />}
               radios={
                 <>
-                  <Radio size="small" value={HarEtterbetaling.JA}>
+                  <Radio size="small" value={ISvar.JA}>
                     Ja
                   </Radio>
-                  <Radio size="small" value={HarEtterbetaling.NEI}>
+                  <Radio size="small" value={ISvar.NEI}>
                     Nei
                   </Radio>
                 </>
               }
             />
 
-            {watch().harEtterbetaling == HarEtterbetaling.JA && (
+            {watch().harEtterbetaling == ISvar.JA && (
```

Comment:
```suggestion
            {watch('harEtterbetaling') == ISvar.JA && (
```
gjelder forsåvidt andre watches i komponenten, så skal den kanskje være mer effektiv på rerenders enn en watch på alt som så accesses :D

Reply:
Endret på alle [8854352](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5273/commits/8854352d28dacdef48433c15c55e7c96b5ae4f90)